programs/merkle-distributor/src/instructions/admin/new_parent_account.rs

```diff
@@ -0,0 +1,49 @@
+use crate::state::parent_account::ParentAccount;
+use anchor_lang::prelude::*;
+use anchor_spl::token::{Mint, TokenAccount};
+
+#[derive(Accounts)]
+pub struct NewParentAccount<'info> {
```

Comment:
add a function to transfer fund to parent account

Reply:
done with `FundDistributorRoot`